### PR TITLE
fix: test scripts and readme doc about uninstallation

### LINKED ISSUE #315: All the crd should be deleted when uninstall ObO
Need update document and related code:
test/run-e2e-ocp.sh

% for crd in $(oc get crd|grep rhobs|awk '{print $1}'); do echo $crd; oc delete --ignore-not-found customresourcedefinitions $crd; done
% for crd in $(oc get crd|grep rhobs|awk '{print $1}'); do echo $crd; done
alertmanagerconfigs.monitoring.rhobs
alertmanagers.monitoring.rhobs
monitoringstacks.monitoring.rhobs
podmonitors.monitoring.rhobs
probes.monitoring.rhobs
prometheusagents.monitoring.rhobs
prometheuses.monitoring.rhobs
prometheusrules.monitoring.rhobs
scrapeconfigs.monitoring.rhobs
servicemonitors.monitoring.rhobs
thanosqueriers.monitoring.rhobs
thanosrulers.monitoring.rhobs

### PR DESCRIPTION
**Pull request by lihongyan1**
fix #315

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,14 +36,18 @@ It is easier to use the web console to remove the installed operator.
...
-kubectl delete -n openshift-operators  \
+oc delete -n openshift-operators  \
     installplan,subscriptions \
     -l operators.coreos.com/observability-operator.operators=
 
-kubectl delete -f hack/olm
+oc delete -f hack/olm
+
+for crd in $(oc get crd|grep 'monitoring\.rhobs'|awk '{print $1}'); do \
+  oc delete --ignore-not-found customresourcedefinitions $crd; done
```
Comment (simonpasquier) [inline README.md:49]: (suggestion) a bit shorter
```suggestion
kubectl delete crds $(kubectl api-resources --api-group=monitoring.rhobs -o name)
```

File: test/run-e2e-ocp.sh [hunk 2]
```diff
@@ -64,6 +64,8 @@ delete_obo() {
 
 	oc delete -f hack/olm/subscription.yaml || true
 	oc delete -f hack/olm/catalog-src.yaml || true
+	for crd in $(oc get crd|grep 'monitoring\.rhobs'|awk '{print $1}'); do \
```
Comment (simonpasquier) [inline test/run-e2e-ocp.sh:67]: here too

**Review by simonpasquier — APPROVED** [review]
Thanks!

**Comment by simonpasquier** [comment]
Just one small fix to make the lint check green ;)

```
 In ./test/run-e2e-ocp.sh line 67:
make: *** [Makefile:38: lint-shell] Error 123
	oc delete crds $(oc api-resources --api-group=monitoring.rhobs -o name)
                       ^-- SC2046 (warning): Quote this to prevent word splitting.
````

File: README.md [hunk 3]
```diff
@@ -36,14 +36,18 @@ It is easier to use the web console to remove the installed operator.
 Instructions below removes all traces of what was setup in the previous step
 including removing the catalog.
 ```
-kubectl delete -n operators csv \
+oc delete -n operators csv \
```
[the remark this answers is not in the document]
Comment (danielmellado) [reply README.md:39]: totally +1, it'll help being specific to openshift, although `kubectl` would work as well :D

File: test/run-e2e-ocp.sh [hunk 4]
```diff
@@ -64,6 +64,7 @@ delete_obo() {
 
 	oc delete -f hack/olm/subscription.yaml || true
 	oc delete -f hack/olm/catalog-src.yaml || true
+	oc delete crds $(oc api-resources --api-group=monitoring.rhobs -o name)
```
Comment (danielmellado) [inline test/run-e2e-ocp.sh:67]: As Simon comments, please change this to let the linter go green, thanks!

```suggestion
	oc delete crds "$(oc api-resources --api-group=monitoring.rhobs -o name)"
```

**Comment by lihongyan1** [comment]
@simonpasquier @danielmellado Please help review and approve

**Review by jan--f — APPROVED** [review]

**Pull request merged**